requirements.txt

```
@@ -0,0 +1,13 @@
+
+# List here all the required packages of the project's development environment.
+
+pip
+wheel
+watchdog
+tox
+coverage
+Sphinx
+numpy
+xarray
+netCDF4
+pysteps
```

```suggestion
pysteps>=1.4,<2.0
```

I suggest to explicitly state the minimum (1.4 introduced the plugins) and maximum (2.0 will likely change the interface to the importers) versions here.